### PR TITLE
Kdr draftmenu

### PR DESCRIPTION
**Pull request by kdreese**
Drafting screen is here 😤

### CODE REVIEW

**Review by admoore0 — APPROVED** [review]

File: src/autoload/global.gd [hunk 1]
```diff
@@ -5,6 +5,35 @@ const MAX_VOLUME_DB = -6.0
 
 
 var animation_speed: float = 0.25
+var attack_cards := {}		# Dictionary[int, Array[CardData]]
+var defense_cards := {}		# Dictionary[int, Array[CardData]]
+var curr_stage: int = 0
```
Comment (thearst3rd) [inline src/autoload/global.gd:10]: This should probably be stored in game rather than global

File: src/states/game/game.gd [hunk 2]
```diff
@@ -15,7 +16,7 @@ const BLUE_CASTLE_DOOR = Vector2(480, 80)
 
 var enemy_moves: Array[Dictionary]
 var cards: Array[Resource]
-var num_rounds: int = 0
+var curr_round: int = 0
```
Comment (thearst3rd) [inline src/states/game/game.gd:19]: Oh, it is lol. Remove the one in global

Comment (thearst3rd) [reply src/states/game/game.gd:19]: waittttt this is different I think idk maybe

**Review by thearst3rd — APPROVED** [review]
image

File: src/states/game/game.gd (hunk 2 above)
Comment (kdreese) [reply src/states/game/game.gd:19]: `curr_rounds` is separate from `curr_stage`

**Pull request merged**